prometheus-node-exporter-lua: add hostapd exporter

blocktrron opened the pull request:
Maintainer: @champtar 
Compile tested: ath79 / Ubiquiti UniFi AC Lite
Run tested: ath79 / Ubiquiti UniFi AC Lite

Description:

This exporter exposes information of the connected stations acquired
from hostapd. These contain additional information compared to the
existing station exporter, however they require a full build of hostapd
/ wpad.

champtar commented:
Hello @blocktrron 
If you want to shorten a bit all the names you can `s/hostapd_wifi_station/hostapd_station/`, else LGTM

blocktrron commented:
@champtar altered naming

champtar commented:
Having a second look, the comment about needing a full build of hostapd / wpad is only in the commit message, can you had it maybe in the description of the package ?
Also open question: do we need to bump the package version for the buildbots or they will rebuild everything anyway ?

blocktrron commented:
@champtar 

They should always rebuild the full packages set afaik. I've updated the package version anyway.

Added notice about the hostapd / wpa_supplicant requirements.

champtar commented:
@blocktrron you forgot to push ?

blocktrron commented:
Yup, derped. Fixed.

champtar commented:
Thanks @blocktrron !
